frontend/src/employee-frontend/components/messages/api.ts

```diff
@@ -127,3 +131,63 @@ export async function getSenderOptions(
     .then((res) => Success.of(res.data))
     .catch((e) => Failure.fromError(e))
 }
+
+export async function getAccounts(): Promise<Result<MessageAccount[]>> {
```

Comment:
We updated the return type to `EnrichedMessageAccount`, so this will be outdated soon. But it probably makes sense to make the update it in the branch that I'm working on atm.